src/app/shared/components/workshop-card/workshop-card.component.html

```diff
@@ -83,11 +83,13 @@
   <p *ngIf="!isHorizontalView" class="card_text">
     <mat-icon class="card_icon">star</mat-icon> {{ workshopData.rating }}
   </p>
-  <div class="card_text">
+
+  <a [routerLink]="['/details/provider',workshop?.providerId ]" fxLayout="row" fxLayoutAlign="start center">
```

Comment:
same question here. if we display workshop card does it mean that we have those ids?